Description: udev rules for the Librem Key
 udev rules for the Librem Key
 .
 libnitrokey (3.5-0pureos1) byzantium; urgency=medium
 .
   * Updating with support for Purism's Librem Key.
Author: Jeremiah C. Foster <jeremiah.foster@puri.sm>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2020-05-27

--- /dev/null
+++ libnitrokey-3.5/data/60-librem-key.rules
@@ -0,0 +1,3 @@
+# my librem key
+SUBSYSTEM=="usb", ATTR{idVendor}=="316d", ATTR{idProduct}=="4c4b", ENV{ID_SMARTCARD_READER}="1", ENV{ID_SMARTCARD_READER_DRIVER}="gnupg"
+
